### PR TITLE
fixing bug where all default to upcoming

### PR DESCRIPTION
**Pull request by philipdouble**

### CODE REVIEW

File: src/schemas/engagement.ts [hunk 1]
```diff
@@ -267,6 +267,7 @@ export const getEngagementStatus = (
   if (!engagement) {
     return null;
   }
+  // console.log(engagement);
```
Comment (oybed) [inline src/schemas/engagement.ts:270]: did you mean to leave this one here?

Comment (philipdouble) [reply src/schemas/engagement.ts]: removed

Comment (philipdouble) [reply src/schemas/engagement.ts]: PS - tnx - i like tidiness!

**Review by oybed — APPROVED** [review]
LGTM

**Pull request merged**